jwql/website/jwql_proj/jinja2.py

```diff
@@ -19,12 +19,12 @@
 from django.utils import timezone
 from django.template.defaultfilters import date
 from django.contrib.staticfiles.storage import staticfiles_storage
-from jinja2 import Environment, lexer, nodes
+from jinja2 import Environment, lexer, nodes, select_autoescape
```

Comment:
I am good with that. I was wondering how this would effect the webapp since we are passing `kwargs`. We can just pass it if we are confident in `**options` and how they are passed 👍